reducers/graphData.js

```diff
@@ -31,24 +36,30 @@ function combineTwoLines(primaryGraph, secondaryGraph) {
     });
     entry = mapIter.next().value;
   }
-
+  // console.log ('res', res)
   res.sort((a, b) => (new Date(a.date) <= new Date(b.date) ? -1 : 1));
   // console.log(res);
 
   return res;
 }
 
-export default function (state = [], action) {
+function retrieveRelative () {
```

Comment:
@JeffreyStocker : What's the plan with this function? I'm cool with it, just curious though since it's empty.